modules/ROOT/pages/ch2-custom-alerts.adoc

```diff
@@ -1,136 +1,26 @@
-
-=== Automatic Notifications
-
-*THIS CONTENT IS FROM CH 1.0*
-
-For applications deployed to CloudHub 2.0, some events automatically produce a notification:
-
-[%header,cols="30a,70a"]
-|===
-|Event |Message
-| Worker will restart | A worker `<ip>` has become unresponsive. Restarting the worker.
-| Worker has restarted | Restart SUCCEEDED for worker `<ip>`
-| Worker restart failed | Restart FAILED for worker `<ip>`
-| Maximum Restarts attempted for a worker| Maximum number of restart attempts reached
-| Worker needs restarting, but restarts are disabled | A worker `<ip>` has become unresponsive. Restarts are disabled.
-|===
-
-=== Notification Retention
-
-Anypoint Platform retains notifications up to the following limits, whichever occurs first:
-
-* 30 days
-* Maximum of 1000 notifications per application
-+
-If the number of notifications exceeds the maximum,
-CloudHub 2.0 deletes the oldest notification.
-+
-Each notification can contain up to 10,000 characters.
-
-
-== Alerts
-
-You can create standard alerts in Runtime Manager for applications deployed to CloudHub 2.0.
+_Alerts_ indicate whether an app is behaving as expected. You can create alerts in Runtime Manager for applications deployed to CloudHub 2.0.
 See https://docs.mulesoft.com/runtime-manager/alerts-on-runtime-manager[Alerts^] (Link out from Beta docs).
 
-If you create custom notifications, you can create custom alerts to send emails when 
-those notifications occur.
-
 === Rate Limits on Alerts
 
-Anypoint Platform enforces rate limits that determine how often an alert is triggered and prevent excessive emails from being sent.
-The following factors determine the rate limit:
+Anypoint Platform enforces rate limits that determine how often an alert is triggered and prevent excessive emails from being sent. The rate limit is one alert notification email sent every 2.5 seconds per organization.
 
-* CloudHub 2.0 custom application alerts
-+
-When using CloudHub Connector to trigger notifications and alerts,
-the rate limit is five custom application alerts per IP address, per second.
-This limit is enforced by IP address.
-If an application uses multiple workers, this limit is multiplied by the number of workers
-assigned to the application.
-+
-Custom notifications that trigger emails are also subject to the rate limit on email notifications.
-* Alert emails
-+
-For both standard and custom alerts, the rate limit is one alert notification email sent every 2.5 seconds per organization.
 This limit is enforced by a combination of organization, environment, alert (name or ID), and application.
 
-
-
-
-== Custom Notification and Alert Workflow
-
-. <<configure-custom-notification,Configure a custom notification in CloudHub Connector.>>
-. <<configure-custom-app-alerts,Bind the custom notification to an alert in Runtime Manager.>>
-. xref:ch2-deploy.adoc[].
-
 === Prerequisites
```

Comment:
And here. Also, can you change this to Before You Begin?

Reply:
```suggestion
== Before You Begin
```